jobs/e2e-test-cypress.groovy

```diff
@@ -73,66 +71,56 @@ Defaults to GIT_REVISION.""",
 """,
    ["all", "deploy", "nightly"]
 
-
 ).apply()
 
 def _setupWebapp() {
-   GIT_SHA1 = kaGit.resolveCommitish("git@github.com:Khan/webapp",
-                                     params.GIT_REVISION);
-   
-   kaGit.safeSyncToOrigin("git@github.com:Khan/webapp", GIT_SHA1);
-
-   dir("webapp") {
-      sh("make clean_pyc");
+   kaGit.safeSyncToOrigin("git@github.com:Khan/webapp", params.CYPRESS_GIT_REVISION);
 
+   dir("webapp/services/static") {
+      sh("yarn install")
    }
 }
 
 def runLamdaTest() {
-    _setupWebapp();
-
-    // We need login creds for LambdaTest Cli
-    def lt_username = sh(script: """\
-        keeper --config ${exec.shellEscape("${HOME}/.keeper-config.json")} \
-        get qvYpo_KnpCiLBN69fYpEYA --format json | jq -r .login\
-        """, returnStdout:true).trim();
-    def lt_access_key = sh(script: """\
-        keeper --config ${exec.shellEscape("${HOME}/.keeper-config.json")} \
-        get qvYpo_KnpCiLBN69fYpEYA --format json | jq -r .password\
-        """, returnStdout:true).trim();
-    
-    def runLambdaTestArgs = ["--cy=\"--config baseUrl=${params.URL}\", retries=${params.TEST_RETRIES}",
-                             "--bn",
-                             "e2e-test # (${params.URL}: ${params.REVISION_DESCRIPTION}) @${params.DEPLOYER_USERNAME}",
-                             "-p",
-                             "${params.NUM_WORKER_MACHINES}"
-    ];
-    
-    try {
-        dir('webapp/services/static') {
-            
-            sh("git checkout feature/cypress");
-            sh("yarn install")
-
-            withEnv(["LT_USERNAME=${lt_username}",
-                     "LT_ACCESS_KEY=${lt_access_key}"]) {
-                sh("yarn lambdatest ${exec.shellEscapeList(runLambdaTestArgs)}")     
-            }
-        }
-    } catch (e) {
-        throw new TestFailed("LambdaTest build failed!");
-    }
+   _setupWebapp();
```

Comment:
Normally we do this as a separate stage, so you'd have
```
stage("Sync webapp") {
   _setupWebapp();
}
stage("Run e2e tests") {
   ...
}
````
in your main section.  That just lets us get more accurate timing information on how long it takes to actually run the lambda test.